compilation/src/main/scala/scala/tools/nsc/ScalacBenchmark.scala

```diff
@@ -6,23 +6,30 @@ import java.nio.file.attribute.BasicFileAttributes
 import java.util.concurrent.TimeUnit
 import java.util.stream.Collectors
 
-import org.openjdk.jmh.annotations._
 import org.openjdk.jmh.annotations.Mode._
+import org.openjdk.jmh.annotations._
 
 @State(Scope.Benchmark)
 class ScalacBenchmark {
-  @Param(value = Array[String]())
+  @Param(value = Array())
   var source: String = _
+
   @Param(value = Array(""))
   var extraArgs: String = _
+
+  // This parameter is set by ScalacBenchmarkRunner / UploadingRunner based on the Scala version.
+  // When running the benchmark directly the "latest" symlink is used.
+  @Param(value = Array("latest"))
+  var corpusVersion: String = _
+
   var driver: Driver = _
 
   def compileImpl(): Unit = {
     val compilerArgs =
       if (source.startsWith("@")) Array(source)
       else {
         import scala.collection.JavaConverters._
-        val allFiles = Files.walk(findSourceDir).collect(Collectors.toList[Path]).asScala.toList
+        val allFiles = Files.walk(findSourceDir, FileVisitOption.FOLLOW_LINKS).collect(Collectors.toList[Path]).asScala.toList
```

Comment:
This might not be windows friendly. That isn't a blocker for now, but maybe to support contributors on that platform, and our future plans to setup a Windows benchmark server, we should emulate the symlink.